recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -0,0 +1,18 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+SRC_URI += " \
+	file://0001-wwan-fix-provision-of-IPv6-addresses.patch \
+	file://NetworkManager.conf \
+"
+
+# NOTE: the patch is only needed with NetworkManager 1.36, in 1.38 and above it
+#       is already there
+
+PACKAGECONFIG:append = " modemmanager"
+PACKAGECONFIG:remove = " dnsmasq"
+
+do_install:append() {
+    install -m 0600 ${WORKDIR}/NetworkManager.conf ${D}${sysconfdir}/NetworkManager/
+}
+
+ALTERNATIVE_PRIORITY[resolv-conf] = "10"
```

Comment:
does this mean, if i don't use wwan i have no dns resolver?

Reply:
as long as wwan is not connected Ethernet settings would be used.
so, if eth0 gets dns and gateway settings via dhcp everything works. that is exactly why I needed to deactivate gateway settings in pfsense for rpi4-onect-lab duts.
not sure I understood your question correctly, though.